lib/command.js

```diff
@@ -735,12 +740,22 @@ Expecting one of '${allowedValues.join("', '")}'`);
    */
   passThroughOptions(passThrough = true) {
     this._passThroughOptions = !!passThrough;
-    if (!!this.parent && passThrough && !this.parent._enablePositionalOptions) {
-      throw new Error('passThroughOptions can not be used without turning on enablePositionalOptions for parent command(s)');
-    }
+    this._checkForIllegalPassThroughOptions(this, this.parent);
     return this;
   }
 
+  /**
+   * @param {Command} command
+   * @param {Command | null} parent
+   * @api private
+   */
+
+  _checkForIllegalPassThroughOptions(command, parent) {
+    if (parent && command._passThroughOptions && !parent._enablePositionalOptions) {
+      throw new Error(`passThroughOptions cannot be used for '${command._name}' without turning on enablePositionalOptions for parent command${parent._name ? ` '${parent._name}'` : ''}`);
```

Comment:
I was assuming that the throw would include the line number so easy enough to track down, but I do like adding the subcommand name. Easier to jump into fixing.

Adding the parent command name is lower value and makes the code harder to read.
```suggestion
      throw new Error(`passThroughOptions cannot be used for '${command._name}' without turning on enablePositionalOptions for parent command(s)`);
```

Reply:
I agree the parent command name is not important here.

I have actually been working on a proof of concept PR introducing support for multiple parent commands. But before it is added, I wonder what exactly is the purpose of the `(s)` part at the end of the error message?

Reply:
For safety, positional options should be turned on all the way up the tree. In general I expect it will be turned on at the root, and that intermediate levels won't have options, so unlikely to have a practical effect, and not attempting to robustly enforce. But I thought it was worth adding the `(s)` to hint at the multiple levels.